### ISSUE TITLE
Getting latitude and longitude values for a device

### ISSUE THREAD

**Issue by kelvintaku** [vitalidze/traccar-web#1227]
Hello guys, im new on this kind of proyect and im trying to get only latitude and longitude value from a device to put in a message, how can i get those? and thanks in advance

**Comment by vitalidze** [comment]
Not sure what is your question about. Do you use traccar system for receiving data from your device?

**Comment by kelvintaku** [comment]
Yeah, im using traccar system but im trying to add an option in the context menu of devices that links to google maps web for that device's latitude and longitude, i got to add the menu item, but im trying to figue how to get latitude and longitude

**Comment by vitalidze** [comment]
If you are using the traccar web UI mod (this project), then you can get it via API: http://traccar.litvak.su/features/rest-api.html

**Comment by kelvintaku** [comment]
Oh thanks, ill test it out, so i can get latitude and longitude values for a device witht this api, right?

**Comment by vitalidze** [comment]
Yes, thats correct

чт, 4 июл. 2019 г., 18:41 kelvintaku <notifications@github.com>:
